proposals/2025/Add Federal Fields.md

```diff
@@ -0,0 +1,117 @@
+# Add Federal Fields
+
+## Overview
+
+This proposal is to add new elements under the `FederalBuilding` element:
+
+* `auc:Agency` - this field already exists. It is a text field and should probably stay that way to map to different tools. For the CERL use case, this maps to "DOD"
+* `auc:SubAgency` - Sub Agency according to agency's organizational structure. For CERL's workflow, this would be "Army".
+* `auc:SubAgencySubLevel1` - Sub Agency Sub Level 1 represents another level down in the agency's organizational structure. For CERL's workflow, this would be the "Accountable Organization".
+* `auc:SubAgencySubLevel2` - Sub Agency Sub Level 2 represents another level down in the agency's organization structure. For CERL's workflow this would be the "Accountable Sub Organization".
+* `Facility` - Federal Facility represents another level down in the agency's organization structure. For CERL's workflow, this would be the "Installation".
```

Comment:
Would this be a duplicated name with the existing `Facility` element?

Reply:
good catch, I'll rename it to "FederalFacility"